docs/ocis/deployment/ocis_traefik.md

```diff
@@ -11,118 +11,92 @@ geekdocFilePath: ocis_traefik.md
 See also [example server setup]({{< ref "preparing_server.md" >}})
 
 
 ### Install oCIS and Traefik
 
-The application stack contains two containers. The first one is a traefik proxy which is terminating ssl and forwards the requests to the internal docker network. Additional, traefik is creating a certificate that is stored in `acme.json` in the folder `letsencrypt` inside the users home directory.
-The second one is th ocis server which is exposing the webservice on port 9200 to traefik.
-
-* Clone ocis repository
+* Clone oCIS repository
 
   `git clone https://github.com/owncloud/ocis.git`
 
-* Copy example folder to /opt
-
-  `cp deployment/examples/ocis_traefik /opt/`
+* Go to the deployment example
 
-* Overwrite OCIS_DOMAIN in .env with your.domain.com
+  `cd ocis/deployment/examples/ocis_traefik`
 
-  `sed -i 's/ocis.domain.com/your.domain.com/g' /opt/ocis_traefik/.env`
+* Open the `.env` file in a text editor
+  The file by default looks like this:
+  ```bash
+  # If you're on a internet facing server please comment out following line.
+  # It skips certificate validation for various parts of oCIS and is needed if you use self signed certificates.
+  INSECURE=true
 
-* Overwrite redirect uri with your.domain.com in identifier-registration.yml
+  ### Traefik settings ###
+  # Domain of Traefik, where you can find the dashboard. Defaults to "traefik.owncloud.test"
+  TRAEFIK_DOMAIN=
+  # Basic authentication for the dashboard. Defaults to user "admin" and password "admin"
+  TRAEFIK_BASIC_AUTH_USERS=
+  # Email address for obtaining LetsEncrypt certificates, needs only be changed if this is a public facing server
+  TRAEFIK_ACME_MAIL=
 
-  `sed -i 's/ocis.domain.com/your.domain.com/g' /opt/ocis_traefik/config/identifier-registration.yml`
+  ### oCIS settings ###
+  # oCIS version. Defaults to "latest"
+  OCIS_DOCKER_TAG=
+  # Domain of oCIS, where you can find the frontend. Defaults to "ocis.owncloud.test"
+  OCIS_DOMAIN=
+  ```
 
-* Change into deployment folder
+  You are installing oCIS on a server and Traefik will obtain valid certificates for you so please remove `INSECURE=true` or set it to `false`.
 
-  `cd /opt/ocis_traefik`
+  Set your domain for the Traefik dasboard in `TRAEFIK_DOMAIN=` eg. `TRAEFIK_DOMAIN=traefik.owncloud.test`.
 
-* Start application stack
+  The Traefik dasboard is secured by basic auth. Default credentials are the user `admin` with the password `admin`. To set your own credentials, generate a htpasswd (eg. by using [an online tool](https://htpasswdgenerator.de/) or a cli tool).
 
-  `docker-compose up -d`
+  Traefik will issue certificates with LetsEncrypt and therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
 
-### Configuration
+  oCIS will by default started in the `latest` version. If you want to start a specific version of oCIS set the version to `OCIS_DOCKER_TAG=`. Available versions can be found on [Docker Hub](https://hub.docker.com/r/owncloud/ocis/tags?page=1&ordering=last_updated).
```

Comment:
Proposal: `By default oCIS will be started in the latest version.` instead of `oCIS will by default started in the latest version.`